e2e-test/trace/build.gradle

```diff
@@ -97,16 +98,17 @@ task pullDataPrepperDockerImage(type: DockerPullImage) {
 }
 
 def createDataPrepperDockerContainerFromPullImage(final String taskBaseName, final String dataPrepperName, final int grpcPort,
-                                                  final int serverPort, final String pipelineConfigYAML) {
+                                                  final int serverPort, final int peerForwarderPort,
+                                                  final String pipelineConfigYAML, final String dataPrepperConfigYAML) {
     return tasks.create("create${taskBaseName}", DockerCreateContainer) {
         dependsOn createDataPrepperNetwork
         dependsOn pullDataPrepperDockerImage
         containerName = dataPrepperName
-        hostConfig.portBindings = [String.format('%d:21890', grpcPort), String.format('%d:4900', serverPort)]
-        exposePorts('tcp', [21890, 4900])
+        hostConfig.portBindings = [String.format('%d:21890', grpcPort), String.format('%d:4900', serverPort), String.format('%d:4994', peerForwarderPort)]
+        exposePorts('tcp', [21890, 4900, 4994])
         hostConfig.network = createDataPrepperNetwork.getNetworkName()
         hostConfig.binds = [(project.file(pipelineConfigYAML).toString()):"/usr/share/data-prepper/pipelines.yaml",
-                            (project.file("src/integrationTest/resources/data_prepper.yml").toString()):"/usr/share/data-prepper/data-prepper-config.yaml"]
+                            (project.file(dataPrepperConfigYAML).toString()):"/usr/share/data-prepper/data-prepper-config.yaml"]
```

Comment:
The paths to the two yaml files are not complete. 

I think a prefix of `src/integrationTest/resources/` is needed either here in this function, like:
```
project.file("src/integrationTest/resources/${pipelineConfigYAML}")
```
 or when you pass in the arguments - pass in `"src/integrationTest/resources/${RAW_SPAN_PIPELINE_LATEST_RELEASE_YAML}"` instead of `"${RAW_SPAN_PIPELINE_LATEST_RELEASE_YAML}"`

Reply:
Good catch. 
The compatibility test won't work till we release 2.0. I disabled this workflow till the release. You can see it here
https://github.com/opensearch-project/data-prepper/actions/workflows/data-prepper-trace-analytics-raw-span-compatibility-e2e-tests.yml